wiki/Beatmap/Pattern/common/double/en.md

```diff
@@ -0,0 +1,20 @@
+---
+stub: true
+tags:
+  - doubles
+---
+
+# Doubles
+
+A **Double** pattern is made when 2 [hit objects](/wiki/Gameplay/Hit_object) occur after each other in a 1/2 or faster rythm spacing.
+
+![](img/double.jpg "Visual representation of a double in the osu! gamemode.")
+
+In the [osu!](/wiki/Game_mode/osu!) [gamemode](/wiki/Game_mode), doubles are when 2 [hit circles](/wiki/Gameplay/Hit_object/Hit_circle) are [stacked](/wiki/Beatmapping/Mapping_techniques/Stack) on eachother.
```

Comment:
^, i don't think this is clear enough

```suggestion
In the [osu!](/wiki/Game_mode/osu!) [gamemode](/wiki/Game_mode), doubles are when two [hit circles](/wiki/Gameplay/Hit_object/Hit_circle) are [stacked](/wiki/Beatmapping/Mapping_techniques/Stack) on top of each other.
```
my suggestion doesn't resolve it entirely in being more clear but it should be a step in the right direction

Reply:
Will change it after the respository will be done cloning on my pc, so i will be able to make changes

Reply:
> Will change it after the respository will be done cloning on my pc, so i will be able to make changes

commit by pressing `Commit suggestion` on the right